iptables/iptables.go

```diff
@@ -173,7 +175,7 @@ func (ipt *IPTables) runWithOutput(args []string, stdout io.Writer) error {
 	args = append([]string{ipt.path}, args...)
 	if ipt.hasWait {
```

Comment:
I don't get it - surely 1.6.0 should pass `hasWait` and so we never reach this internal flock?